src/app/learn/get-started/advanced/sticky-settings/index.html

```diff
@@ -81,4 +81,22 @@
     Currently, the <a routerLink="/components/grid">grid component</a> and <a routerLink="/components/tile">tile component</a> both support a <sky-code>settingsKey</sky-code> property for sticky settings. When you provide this property, the components automatically call the <sky-code>SkyUIConfigService</sky-code> stub to retrieve stored settings during appropriate actions. This is why you provide your own implementation as shown above.
   </p>
 
+  <stache-page-anchor>
+    Unit tests
+  </stache-page-anchor>
+  <p>
+    To avoid external dependencies, inject the base class when you provide a custom <sky-code>SkyUIConfigService</sky-code>.
```

Comment:
I think this needs to be clarified or expounded slightly.

At a minimum, I would suggest it read as (directly from @Blackbaud-PaulCrowder):

```
You can either inject the default `SkyUIConfigService` back in in your `TestBed` configuration, or provide your own class if you need to return fake data.
```